src/toolbar.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import { startHighlight } from './highlight';
+
+
+const toolbarStyle: React.HTMLAttributes<HTMLDivElement>["style"] = {
+  display: 'flex',
+  flexDirection: 'column',
+  position: 'fixed',
+  top: '100px',
+  right: '0px',
+  padding: '1px'
```

Comment:
Styling generally looks good, just try to use em (or some other relative measure, like % or vw/vh) instead of px. I find that 1px padding/borders are fine though.

Reply:
4 other occurrences of this: toolbarToggleStyle (2) and toolbarIconStyle (2)